model/src/w3gridmd.F90

```diff
@@ -903,13 +903,13 @@ MODULE W3GRIDMD
 #ifdef W3_PR3
       REAL                    :: WDTHCG, WDTHTH
 #endif
-           LOGICAL :: JGS_TERMINATE_MAXITER = .TRUE.
-           LOGICAL :: JGS_TERMINATE_DIFFERENCE = .TRUE.
-           LOGICAL :: JGS_TERMINATE_NORM = .TRUE.
-           LOGICAL :: JGS_LIMITER = .FALSE.
-           LOGICAL :: JGS_BLOCK_GAUSS_SEIDEL = .TRUE.
-           LOGICAL :: JGS_USE_JACOBI = .TRUE.
-           LOGICAL :: JGS_SOURCE_NONLINEAR = .FALSE.
+           LOGICAL :: JGS_TERMINATE_MAXITER    =  .TRUE.
+           LOGICAL :: JGS_TERMINATE_DIFFERENCE =  .TRUE.
+           LOGICAL :: JGS_TERMINATE_NORM       = .FALSE.
+           LOGICAL :: JGS_LIMITER              = .FALSE.
+           LOGICAL :: JGS_BLOCK_GAUSS_SEIDEL   = .TRUE.
+           LOGICAL :: JGS_USE_JACOBI           = .TRUE.
+           LOGICAL :: JGS_SOURCE_NONLINEAR     = .FALSE.
            LOGICAL :: UGOBCAUTO = .FALSE.
            LOGICAL :: UGBCCFL   = .FALSE.
            LOGICAL :: EXPFSN    = .TRUE.
```

Comment:
This is the value of EXPFSN that I was referring to in the below comment.

Reply:
Except that UGBCCFL   = .TRUE., the defaults are fine like that. Now we can keep it as it is in develop.

Reply:
I think that we keep the default settings as in develop.